### PR TITLE
feat: add timeout for server stop as method param

### PR DESCRIPTION
**Pull request by LykashevichAndrei**

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
I am not sure what you are trying to achieve by this PR. Active connections close timeout is currently [hardcoded](https://github.com/appium/appium/blob/6a8f26a9cbf01a7c9181c90bb999d332a343d94d/packages/base-driver/lib/express/server.js#L34) on the server side and won't be affected by the process timeout.

**Comment by LykashevichAndrei** [comment]
https://github.com/appium/python-client/issues/1027

at least for now i will not see error on test postconditions that appium server was not closed

however this is a good catch and i will review appium server code and may be provide another PR to avoid this hardcode

File: appium/webdriver/appium_service.py [hunk 1]
```diff
@@ -222,20 +222,23 @@ def start(self, **kwargs: Any) -> sp.Popen:
             raise AppiumServiceError(error_msg)
         return self._process
 
-    def stop(self) -> bool:
+    def stop(self, timeout: int = 5) -> bool:
```
Comment (mykola-mokhnach) [inline appium/webdriver/appium_service.py:225]: python timedeltas are represented as float seconds, not integers

**Comment by mykola-mokhnach** [comment]
Please rename the PR to comply with the https://www.conventionalcommits.org/en/v1.0.0/ standard

File: appium/webdriver/appium_service.py (hunk 1 above)
Comment (KazuCocoa) [inline appium/webdriver/appium_service.py:225]:
```suggestion
    def stop(self, timeout: float = 5) -> bool:
```

**Comment by mykola-mokhnach** [comment]
Closed in favour of https://github.com/appium/python-client/pull/1038

**Pull request closed without merging**